src/app/contextBroker/contextBroker.cpp

```diff
@@ -602,15 +605,19 @@ PaArgument paArgs[] =
 
 
 
-#define API_V2                                                                                                   \
-  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      entryPointsTreat                          }, \
-  { "*",      EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      badVerbAllFour                            }, \
-  { "GET",    ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      getEntities                               }, \
-  { "*",      ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      badVerbGetOnly                            }, \
-  { "GET",    IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     getEntity                                 }, \
-  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     badVerbGetOnly                            }, \
-  { "GET",    IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, getEntityAttribute                        }, \
-  { "*",      IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, badVerbGetOnly                            }
+#define API_V2                                                                                     \
+  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,       entryPointsTreat           }, \
```

Comment:
Currently https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/user/structured_attribute_valued.md says:

> Note that in order to align JSON representations, the final "leaf" elements of the structured attribute values after traversing all vectors and key-maps are always considered as strings.Thus, take into account that although you can use for instance a JSON integer as a field value in updates (such as {"x": 3 }),you will retrieve a string in queries and notifications (i.e. { "x": "3"}).

We should add a comment in this file in order to explain that with /v2 operations now is possible to have numbers and booleans (in general, from now on, as the doc is now in the repository, we should consider code+test+doc modifications in the same PR).